src/server/hotkey_collector.cpp

```diff
@@ -33,5 +53,40 @@ void hotkey_collector::capture_raw_key(const dsn::blob &raw_key, int64_t weight)
 
 void hotkey_collector::capture_hash_key(const dsn::blob &hash_key, int64_t weight) {}
 
+bool hotkey_collector::start_detect(std::string &err_hint)
+{
+    auto now_state = _state.load();
+    switch (_state.load()) {
+    case hotkey_collector_state::COARSE_DETECTING:
+    case hotkey_collector_state::FINE_DETECTING:
+        err_hint = fmt::format("still detecting {} hotkey, state is {}",
+                               dsn::enum_to_string(_hotkey_type),
+                               enum_to_string(now_state));
```

Comment:
Unify to add or remove `dsn::`

Reply:
change the namespace may lead to compile failed
`dsn::replication::hotkey_type::type` and `pegasus::hotkey_collector_state` are defined in different namespaces